# Auto-generated diff of uncommitted changes in the Cylc workflow repository:
#   /oscar/home/dwatkin2/ice-floe-tracker-pipeline/config/cylc_hpc
diff --git a/config/cylc_hpc/flow_template_hpc.j2 b/config/cylc_hpc/flow_template_hpc.j2
index 0c5bab0..b8e4f98 100644
--- a/config/cylc_hpc/flow_template_hpc.j2
+++ b/config/cylc_hpc/flow_template_hpc.j2
@@ -19,8 +19,8 @@
             crs = {{ crs }} # epsg3413 for polar stereographic: left_x@top_y@right_x@lower_y
                              # wgs84 for lat/lon: top_left_lat@top_left_lon@lower_right_lat@lower_right_lon            
             bounding_box = {{ bounding_box }}
-            centroid_lat = {{ centroid_lat }}
-            centroid_lon = {{ centroid_lon }}
+            centroid_lat = {{ center_lat }}
+            centroid_lon = {{ center_lon }}
             minfloearea = {{ minfloearea }}
             maxfloearea = {{ maxfloearea }}
             project_dir = "~/ice-floe-tracker-pipeline"
